requirements.txt

```diff
@@ -1 +1,2 @@
--e git+https://github.com/poldracklab/niworkflows.git@1c00a2490b7c5c9fd2c5d9afa7b6861d4ccecd08#egg=niworkflows
+-e git+https://github.com/effigies/nipype.git@a5fe6bb#egg=nipype
+-e git+https://github.com/effigies/niworkflows.git@7051b39#egg=niworkflows
```

Comment:
Could we instead temporarily remove '>=0.13.0-rc1' from info (in both niworkflows and fmriprep) and switch those to two lines in requirements.txt to point to main niworkflows and nipype repositories? It will make it more straightforward for others to build on top.